src/ocm/services/ClustersService.ts

```diff
@@ -4,8 +4,11 @@ import InfraEnvsService from './InfraEnvsService';
 import { AI_UI_TAG, Cluster, Host, V2ClusterUpdateParams } from '../../common';
 import { ocmClient } from '../api';
 
+const clustersPollingAbortController = new AbortController();
```

Comment:
As discussed, we need to make sure our applications ( `assisted-ui` and `uhc-portal`) use a compatible `axios` version.
Adding a comment to the review as it will most certainly require additional PRs.

Reply:
Ok, I'll try to do the new PRs